README.md

```diff
@@ -107,6 +107,7 @@ Production:
 - [Reading Group. FoundationDB: A Distributed Unbundled Transactional Key Value Store](http://charap.co/reading-group-foundationdb-a-distributed-unbundled-transactional-key-value-store/) (charap.co)
 - [Migrating Snowflake’s Metadata with No Downtime](https://medium.com/snowflake/migrating-snowflakes-metadata-with-no-downtime-ca90604b677c) (medium.com)
 - [Bigblue’s multi-model database on FoundationDB](https://medium.com/bigblue-engineering/bigblues-multi-model-database-on-foundationdb-5c54b8d61942) (medium.com)
+- [Part 2: Migrating Snowflake’s Metadata with FDB Replication Service](https://medium.com/snowflake/part-2-migrating-snowflakes-metadata-with-fdb-replication-service-c3a3bee79904)(medium.com)
```

Comment:
Should we move this directly under the first part?

Reply:
You are right, it will be better, doing